js/index.js

```diff
@@ -106,3 +101,50 @@ messageForm.addEventListener('submit', function(event) {
      event.target.reset();
 
 });
+
+///lesson-15
+
+
+// 'MY_GITHUB_USERNAME'
```

Comment:
This comment does not provide additional information - maybe remove this going forward. However, good use of variables - makes the code reusable which is what you want.